requirements/benchmark.in

```diff
@@ -2,3 +2,4 @@ attrs
 click
 numpy
 scipy
+coverage
```

Comment:
Mmm. Won’t coverage get installed in this virtualenv by virtue of being a Hypothesis dependency?

Reply:
It didn't seem to be. I believe we do not actually install Hypothesis in the virtualenv, just add it to the PYTHONPATH, as that way we can reuse it.